.github/workflows/step-run-tests.yml

```
@@ -123,21 +129,56 @@ jobs:
     - name: Запуск тестирования (Linux)
       if: runner.os == 'Linux'
-      uses: coactions/setup-xvfb@v1
-      with:
-        run: |
-          ibsrv --db-path=file-db --daemon
-          1cv8c /WS "http://localhost:8314" /C"RunUnitTests=yaxunit-config.json" /L ${{ steps.short_locale.outputs.substring }} /VL ${{ inputs.locale }} /DisableStartupDialogs /DisableStartupMessages /DisableUnrecoverableErrorMessage /Out output.log
+      run: |
+        # Запуск ibsrv в фоне
+        ibsrv --db-path=file-db --daemon &
+        sleep 5
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)